applications/crowdloan_frontend_template.md

```diff
@@ -141,17 +142,17 @@ For parts of the community who are less comfortable with the Jamstack approach w
+| 0c. | Article                          | We will publish an article presenting the templates and how to use them.                                                                                                                                                                                                                                                                                                                                                                                     |
+| 1a. | Benchmarking                     | Design research of existing Crowdloan pages and other one pages designs, like Github project pages templates. <br>This will allow us to synthesize viable options of the page designs                                                                                                                                                                                                                                                                        |
+| 1b. | Design Direction Prototype       | Aiming to create as many medium to high-fidelity dd prototypes as possible that allows the Grants team and the community to have an input on the design direction<br>The aim is to have the prototypes at 25%-50% completeness, to see major components/features and the general design direction. This way we don't waste time on dismissed design directions.<br>The designs should follow good practices in general without requiring additional research |
+| 1c. | Repo Setup                       | Repo setup incl. base libraries/frameworks, initial technical documentation. Undesigned base scaffold. Allows the implementation to be simplified by forking                                                                                                                                                                                                                                                                                                 |
+| 2. | Jamstack implementation in Astro | One (1) chosen design direction<br><br>Minimum sections:<br>- Project information<br>- Rewards Schema<br>- Current contributions<br>- Time left in Crowdloan and competition<br>- Contribute CTA<br>- After the Crowdloan<br><br>Implemented as One Page Design                                                                                                                                                                                              |
+| 3. | Webflow implementation           | One (1) chosen design direction<br><br>Minimum sections:<br>- Project information<br>- Rewards Schema<br>- Current contributions<br>- Time left in Crowdloan and competition<br>- Contribute CTA<br>- After the Crowdloan<br>Out of the scope: integrate contribution flow and polkaskan table due to Webflow technical limitations (users of template may still try to implement that on their own way)<br><br>Implemented as One Page Design               |
```

Comment:
Oh, wasn't the issue that you didn't find a suitable CDN? Did you try using the jsdeliv CDN yet [as suggested in my comment](https://github.com/w3f/Grant-Milestone-Delivery/pull/765#issuecomment-1511029621)?

Reply:
Hey, I passed your suggestion to our project team. I've added comment in the [delivery PR](https://github.com/w3f/Grant-Milestone-Delivery/pull/765#issuecomment-1518212162).

Reply:
Okay got it. I've put this PR on hold for the meanwhile.

Reply:
Ok, confirming this due to [this](https://github.com/w3f/Grant-Milestone-Delivery/pull/765#issuecomment-1518212162) and [this](https://github.com/w3f/Grant-Milestone-Delivery/pull/765#pullrequestreview-1397828711) comments.